docs/pages/routing/appearance.mdx

```diff
@@ -14,7 +14,7 @@ There are three major elements that you can use to customize the appearance of y
 
 ## Fonts
 
-Expo Router extends the `expo-splash-screen` API to prevent white flash,
+Expo Router extends the `expo-splash-screen` API to prevent white flash, use it in conjunction with `expo-font` to keep the splash screen visible while fonts are loading:
```

Comment:
```suggestion
Expo Router extends the `expo-splash-screen` API to prevent white flash. Use it in conjunction with `expo-font` to keep the splash screen visible while fonts are loading:
```